database/migrations/2017_04_26_133415_create_trader_user_pivot_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateTraderUserPivotTable extends Migration
```

Comment:
A User is an identity that logs in to manipulate Traders and Vouchers. A User *might* in practice be a Trader but not always; in the systems context is "A person who makes submissions on behalf of one or more traders, and may *be* that trader". Users may well sublass or compose with roles at some point but right now I don't want to couple it too much.